File: Builder/Java/Pizza.java
import java.util.ArrayList;
import java.util.List;

public class Pizza {

    private List<String> PizzaBuild = null;

    Pizza(){
        PizzaBuild = new ArrayList<String>();
    }

    public Pizza setMasa(){
        this.PizzaBuild.add("Masa");
        return Pizza.this;
    }

    public Pizza setSalsa(){
        this.PizzaBuild.add("Salsa");
        return Pizza.this;
    }

    public Pizza setQueso(){
        this.PizzaBuild.add("Queso");
        return Pizza.this;
    }

    public Pizza setPeperoni(){
        this.PizzaBuild.add("Peperoni");
        return Pizza.this;
    }

    public Pizza setPina(){
        this.PizzaBuild.add("Pina");
        return Pizza.this;
    }

    public Pizza setSalchicha(){
        this.PizzaBuild.add("Salchicha");
        return Pizza.this;
    }

    public Pizza setJamon(){
        this.PizzaBuild.add("Jamon");
        return Pizza.this;
    }

    public void PrintPizza(){
        System.out.println(PizzaBuild);
    }
}

File: Builder/Python/Pizza.py
class Pizza:
    PizzaBuild=[]

    def __init__(self):
        self.PizzaBuild=list()

    def setMasa(self):
        self.PizzaBuild.append('Masa')
        return self

    def setSalsa(self):
        self.PizzaBuild.append('Salsa')
        return self

    def setQueso(self):
        self.PizzaBuild.append('Queso')
        return self

    def setPeperoni(self):
        self.PizzaBuild.append('Peperoni')
        return self
    
    def setPina(self):
        self.PizzaBuild.append('Pina')
        return self

    def setSalchicha(self):
        self.PizzaBuild.append('Salchicha')
        return self

    def setJamon(self):
        self.PizzaBuild.append('Jamon')
        return self

    def PrintPizza(self):
        return self.PizzaBuild